packages/xchain-ethereum/__tests__/client.testnet.test.ts

```diff
@@ -555,6 +556,7 @@ describe('Client Test', () => {
     mock_all_api(etherscanUrl, ropstenInfuraUrl, ropstenAlchemyUrl, 'eth_estimateGas', '0x5208')
 
     const gasLimit = await ethClient.estimateApprove({
+      walletIndex: 0,
```

Comment:
I think we can put address: string as a parameter as this function is not signing;